EXCEPTION LIST
Trash paths should be unique, but found multiple entries in the trash index '<dir>/l4/.trash/.index.csv' for the following path: foo

CallStack (from HasCallStack):
  getCallStack, called at src/SafeRm/Effects/MonadCallStack.hs:47:13 in <package>:SafeRm.Effects.MonadCallStack
  throwCS, called at src/SafeRm/Data/Index.hs:226:5 in <package>:SafeRm.Data.Index
  throwIfDuplicates, called at src/SafeRm/Data/Index.hs:119:7 in <package>:SafeRm.Data.Index

LOGS LIST
[2020-05-31 12:00:00][functional.getIndex][Debug] Index path: <dir>/l4/.trash/.index.csv
[2020-05-31 12:00:00][functional.getIndex.readIndex][Debug] Index path: <dir>/l4/.trash/.index.csv
[2020-05-31 12:00:00][functional.getIndex.readIndex.readIndexWithFold][Debug] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "foo"}, originalPath = MkPathI {unPathI = "<dir>/l4/.trash/foo"}, created = MkTimestamp {unTimestamp = 2022-09-28 02:58:33}}
[2020-05-31 12:00:00][functional.getIndex.readIndex.readIndexWithFold][Debug] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "foo"}, originalPath = MkPathI {unPathI = "<dir>/l4/.trash/foo"}, created = MkTimestamp {unTimestamp = 2022-09-28 02:58:33}}
[2020-05-31 12:00:00][functional][Error] Trash paths should be unique, but found multiple entries in the trash index '<dir>/l4/.trash/.index.csv' for the following path: foo

CallStack (from HasCallStack):
  getCallStack, called at src/SafeRm/Effects/MonadCallStack.hs:47:13 in <package>:SafeRm.Effects.MonadCallStack
  throwCS, called at src/SafeRm/Data/Index.hs:226:5 in <package>:SafeRm.Data.Index
  throwIfDuplicates, called at src/SafeRm/Data/Index.hs:119:7 in <package>:SafeRm.Data.Index


